python/lsst/ts/gis/commander.py

```diff
@@ -88,16 +88,24 @@ async def connect(self):
                 self.tunnel_host, self.tunnel_port, timeout=10
             )
             await self.client.connect()
-            self.log.info("Client connected.")
+            if self.client.connected:
+                self.log.info("Client connected.")
 
     async def disconnect(self):
         """Disconnect from the commander.
         sshtunnel is also closed when not in simulation mode.
         """
         if self.connected:
-            await self.client.close()
+            # In pymodbus 3.3, the close method became synchronous
```

Comment:
Since pymodbus now is pinned to `>= 3.5`, is this still needed? If not, please just use the synchronous call and remove the rest.